No external modules should be touching these symbols.
If they are, they are broken.

--- linux-2.6.21.noarch/fs/open.c~	2007-05-27 23:20:25.000000000 -0400
+++ linux-2.6.21.noarch/fs/open.c	2007-05-27 23:20:32.000000000 -0400
@@ -987,7 +987,6 @@ asmlinkage long sys_open(const char __us
 	prevent_tail_call(ret);
 	return ret;
 }
-EXPORT_SYMBOL_GPL(sys_open);
 
 asmlinkage long sys_openat(int dfd, const char __user *filename, int flags,
 			   int mode)
